test/jellyfish_web/integration/server_socket_test.exs

```diff
@@ -85,10 +82,12 @@ defmodule JellyfishWeb.Integration.ServerSocketTest do
 
     Process.exit(room_pid, :kill)
 
-    assert_receive %{
-      "data" => %{"roomId" => ^room_id, "type" => "roomCrashed"},
-      "type" => "controlMessage"
-    }
+    receive do
+      msg when is_binary(msg) ->
+        assert %ControlMessage{
+                 content: {:roomCrashed, %RoomNotification{roomId: ^room_id}}
+               } = ControlMessage.decode(msg)
+    end
```

Comment:
Let's decode message in the ws.ex and assert_receive against ControlMessage structs